environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -695,7 +675,7 @@ objects:
           - mountPath: /var/thanos/store
             name: thanos-store-data
             readOnly: false
-        volumes: []
+        volumes: null
```

Comment:
@metalmatze This comes from upstream, should we keep this?

Reply:
yes!